src/le.py

```diff
@@ -3325,6 +3415,107 @@ def cmd_follow(args):
     request_follow(filename, name, type_opt)
 
 
+def cmd_follow_multilog(args):
+    """
+    Follow the log(s) as defined in string passed to agent with
+    the '--multilog' parameter included - modification of cmd_follow
+    """
+    #### NOTE THAT BEHAVIOUR IS PROMPT USER TO CONTINUE THEN CALL request_follow
+    ####  TO SET UP A LOG WITH THE PATHANME PASSED TO THE AGENT
+
+    if len(args) == 0:
+        die("Error: Specify the file name of the log to follow.")
+    if len(args) > 1:
+        die("Error: Too many arguments.\n"
+            "A common mistake is to use wildcards in path that is being "
+            "expanded by shell. Enclose the path in single quotes to avoid "
+            "expansion.")
+
+    config.load()
+    config.agent_key_required()
+    # FIXME: follow to add logs into local configuration
```

Comment:
When does this planned to be added. If it's a feature then really it should be in a JIRA ticket versus a FIXME  in the code